includes/Elements/Team_Member.php

```diff
@@ -910,9 +913,16 @@ protected function register_controls() {
 	protected function render( ) {
 
         $settings = $this->get_settings_for_display();
-		$team_member_image = $this->get_settings( 'eael_team_member_image' );
+		$team_member_image = $settings['eael_team_member_image'];
+
 		$team_member_image_url = Group_Control_Image_Size::get_attachment_image_src( $team_member_image['id'], 'thumbnail', $settings );
-		if( empty( $team_member_image_url ) ) : $team_member_image_url = $team_member_image['url']; else: $team_member_image_url = $team_member_image_url; endif;
+
+		if ( empty( $team_member_image_url ) ) {
+			$team_member_image_url = $team_member_image['url'];
```

Comment:
Also empty check here (although we are getting this value all the time possibly):
```
$team_member_image_url = $team_member_image['url'] ?? '';
```